xbmc/video/windows/GUIWindowFullScreen.cpp

```diff
@@ -177,6 +177,11 @@ bool CGUIWindowFullScreen::OnAction(const CAction &action)
     }
     break;
 
+  case ACTION_TOGGLE_COMMSKIP:
```

Comment:
player can handle actions by itself. no need to implement a new method to the interface